src/System.Windows.Forms.SourceGenerators/EnumValidationGenerator.cs

```diff
@@ -0,0 +1,178 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CSharp;
+
+namespace System.Windows.Forms.SourceGenerators
+{
+    [Generator]
+    public class EnumValidationGenerator : ISourceGenerator
+    {
+        private const string EnumValidatorStub = @"
+// <auto-generated />
+namespace EnumValidation
```

Comment:
To make it more obvious where this code came from and how to deal with it, perhaps call it:
```suggestion
namespace SourceGenerated
```

The same applies below

Reply:
This is still outstanding.

Reply:
I'm gonna do this, but I don't think it will be very nice to use in practice, and won't be surprised if its changed back later. Why does a developer need to care that the enum validation is generated? I actually think the best namespace for the validator is probably `System.Windows.Forms` so its even easier to find.